pyproject.toml

```diff
@@ -1,17 +1,17 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["pip", "setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "fake-useragent"
+name = "fake_useragent"
```

Comment:
no, we changed this a long time ago to `faker-useragent` on-purpose.

Reply:
see: https://pypi.org/project/fake-useragent/

Reply:
Thanks, restoring it.
I've tested that pip install and all functionalities with pip are not affected by this change.
We had issues with - and _ in wheels with conda, but hopefully this is not related to this name.